projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -66,6 +66,13 @@ function ( $allowed_meta ) {
 
 	if ( \Automattic\Jetpack\Constants::get_constant( 'JETPACK_BETA_BLOCKS' ) ) {
 		add_action( 'the_content', __NAMESPACE__ . '\maybe_get_locked_content' );
+
+		// Close comments on the front-end
+		add_filter( 'comments_open', __NAMESPACE__ . '\maybe_close_comments' );
+		add_filter( 'pings_open', __NAMESPACE__ . '\maybe_close_comments' );
+
+		// Hide existing comments
+		add_filter( 'get_comment', __NAMESPACE__ . '\maybe_gate_existing_comments' );
```

Comment:
Current template doesn't seem to fire `comments_array`, so the following doesn't work

```
add_filter(  'comments_array', '__return_empty_array'  );
```